std/assembly/typedarray.ts

```diff
@@ -1235,14 +1279,35 @@ function SORT<TArray extends ArrayBufferView, T>(
   return array;
 }
 
+// @ts-ignore: decorator
+@inline
+function SLICE<TArray extends ArrayBufferView, T>(
+  array: TArray,
+  start: i32,
+  end: i32
+): TArray {
+  var length = array.length;
+  start  = start < 0 ? max(start + length, 0) : min(start, length);
+  end    = end   < 0 ? max(end   + length, 0) : min(end  , length);
+  length = max(end - start, 0);
+  var slice = changetype<TArray>(__alloc(offsetof<TArray>(), idof<TArray>()));
```

Comment:
Isn't this missing a new buffer (.data, .dataStart) as the target of the memory.copy below?

Reply:
I replace this with `instantiate<T>`